test/src/network.spec.ts

```diff
@@ -486,6 +487,7 @@ describe('network', function () {
 
       const cached: string[] = [];
       page.on('requestservedfromcache', r => {
+        assert(r);
```

Comment:
Revert all changes for test.

Reply:
This is required. See the other comment.